scripts/verify-supabase-flows.mjs

```
@@ -0,0 +1,184 @@
+    }
+
+    if (vacationError) {
+      addCheck('Leitura de vacation_requests', false, vacationError.message);
+    } else {
+      addCheck('Leitura de vacation_requests', true, `${vacations?.length || 0} registros`);
+    }
+
+    if (!scheduleError && !vacationError) {
+      const byDate = new Map();
+      (schedules || []).forEach((s) => {
+        (s.entries || []).forEach((e) => byDate.set(e.date, e));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)